nodes/usb_cam_node.cpp

```diff
@@ -116,6 +116,7 @@ class UsbCamNode
     node_.param("white_balance", white_balance_, 4000);
     node_.param("power_line_frequency", power_line_frequency_, 1);
     node_.param("gamma", gamma_, 50);
+    node_.param("backlight_compensation", backlight_compensation_, 1);
```

Comment:
`backlight_compensation = 1` is the value that you want to change? I would recommend using the `default value=0` here and set 1 at chippy_launch/config/default_camera.yaml

`backlight_compensation (int)    : min=0 max=2 step=1 default=0 value=0`

Reply:
The default parameter that I found was from the terminal using `v4l2-ctl -l`
`backlight_compensation (int)    : min=0 max=2 step=1 default=1 value=0` ? Which is correct?

Reply:
ok, I see, it changes depending on the camera... Good job, it looks good to me.